lib/coverband.rb

```diff
@@ -18,6 +18,7 @@
 require "coverband/collectors/view_tracker"
 require "coverband/reporters/base"
 require "coverband/reporters/console_report"
+require 'coverband/reporters/web'
```

Comment:
so if we require this it is always loaded, aren't we trying to not have this loaded unless the user tries to instantiate it... if we add this here we end up double requiring this file later in the over-riden init method.

Reply:
Yes that should not be there. Great catch.

Reply:
Fixed!